.github/actions/docusaurus-swizzled-warning/index.js

```diff
@@ -0,0 +1,146 @@
+
+    const prCommitRefForFile = file.contents_url.split('ref=')[1]
+    const pkgLockNewJson = await getLargeJsonAtRef(
+      client,
+      github.context.repo.owner,
+      github.context.repo.repo,
+      file.filename,
+      prCommitRefForFile,
+    )
+    const pkgLockOldJson = await getLargeJsonAtRef(
+      client,
+      github.context.repo.owner,
+      github.context.repo.repo,
+      file.filename,
+      'master',
+    )
+
+    const oldVesionModuleKey = findKeyEndingWith(
+      pkgLockOldJson.packages,
+      `node_modules/${packageName}`,
+    )
+    const newVesionModuleKey = findKeyEndingWith(
+      pkgLockNewJson.packages,
+      `node_modules/${packageName}`,
+    )
+    let oldVersion = pkgLockOldJson.packages[oldVesionModuleKey].version
+    let newVersion = pkgLockNewJson.packages[newVesionModuleKey].version
+
+    const oldVesionModuleKeyParent = findKeyEndingWith(
+      pkgLockOldJson.packages,
+      `node_modules/${packageParentName}`,
+    )
+    const newVesionModuleKeyParent = findKeyEndingWith(
+      pkgLockNewJson.packages,
+      `node_modules/${packageParentName}`,
+    )
+    const oldVersionParent =
+      pkgLockOldJson.packages[oldVesionModuleKeyParent].dependencies[
+        packageName
+      ].substring(1)
+    const newVersionParent =
+      pkgLockNewJson.packages[newVesionModuleKeyParent].dependencies[
+        packageName
+      ].substring(1)
+
+    // if parent dependency is higher version then existing
+    // npm install will retrive the newer version from the parent dependency
+    if (oldVersionParent > oldVersion) {
+      oldVersion = oldVersionParent
+    }
+    if (newVersionParent > newVersion) {
+      newVersion = newVersionParent
+    }
+
+    if (newVersion !== oldVersion) {
+      const pkgChangedFiles = await getChangedFilesBetweenTags(
+        client,
+        'cloud-annotations',
+        'docusaurus-openapi',
+        `v${oldVersion}`,
+        `v${newVersion}`,
+      )
+      const changedComponents = overideComponents.filter(
+        componenet =>
+          pkgChangedFiles.filter(
+            path =>
+              path.includes('docusaurus-theme-openapi/src/theme') &&
+              path.includes(componenet),
+          ).length > 0,
+      )
+      const versionReport = `<tbody><tr><td> Old version </td><td> ${oldVersion} </td></tr>
+      <tr><td> New version </td><td> ${newVersion} </td></tr>
+      `
+      const changedComponentsReport = `<tr><td> Overide components changed </td><td> ${changedComponents.join(
+        ', ',
+      )} </td></tr></tbody></table>
+      `
+      const body = messageTemplate + versionReport + changedComponentsReport
+      await client.rest.issues.createComment({
+        owner: github.context.repo.owner,
+        repo: github.context.repo.repo,
+        issue_number: pr.number,
+        body,
+      })
+
+      core.debug('Found changes and posted comment, done.')
+      return
+    }
+
+    core.debug('No changes found, done.')
```

Comment:
Can we make these log statements use `core.info()` instead of `core.debug()` so they always get logged.

Reply:
Sure. This would be easier to understand.
Added with 78551fffbe5c6884663a5309ffb18b98359d78b3